Sources/PublishCLICore/WebsiteRunner.swift

```diff
@@ -12,30 +12,53 @@ internal struct WebsiteRunner {
     let folder: Folder
 
     func run() throws {
-        let generator = WebsiteGenerator(folder: folder)
-        try generator.generate()
-
-        let outputFolder = try resolveOutputFolder()
         let portNumber = 8000
+        
+        try self.generate()
 
         print("""
         🌍 Starting web server at http://localhost:\(portNumber)
-
-        Press any key to stop the server and exit
         """)
 
+        var quit = false
+        
+        while !quit {
+            try startServer(portNumber: portNumber)
+
+            print("""
+
+            Press ⮐  to regenerate the site.
+            Type q⮐  or control-c to quit
+            """)
+            let input = readLine(strippingNewline: true)!
+            switch input.uppercased() {
+            case "Q":
+                quit = true
+            default:
+                try self.generate()
+            }
+        }
+    }
+
+}
+
+private extension WebsiteRunner {
+    func generate() throws {
+        let generator = WebsiteGenerator(folder: folder)
+        try generator.generate()
+    }
+    
+    func startServer(portNumber: Int = 8000) throws {
+        let outputFolder = try resolveOutputFolder()
+
         DispatchQueue.global().async {
             _ = try? shellOut(
                 to: "python -m SimpleHTTPServer \(portNumber)",
                 at: outputFolder.path
             )
         }
-
-        _ = readLine()
     }
-}
 
-private extension WebsiteRunner {
     func resolveOutputFolder() throws -> Folder {
         do { return try folder.subfolder(named: "Output") }
         catch { throw CLIError.outputfolderNotFound }
```

Comment:
```suggestion
        catch { throw CLIError.outputFolderNotFound }
```
This change did not compile for me.
This line has a typo I think. The 'F' in Folder is not capitalized.